### PR TITLE
provide full item availability

### PR DESCRIPTION
**Pull request by tampakis**
It doesn't look the prettiest, but if a mfhd has multiple items, each item's availability gets pulled in on the record show. @axamei we should probably work on improving the side availability panel appearance next week

### CODE REVIEW

**Comment by kevinreiss** [comment]
Hi, I believe that @tpendragon has been shifted the js work over to es6 in the plum project. Is that something that we want to consider for OL too instead of using coffeescript. I think that over time that is going to be the preferred syntax to use over coffeescript.

File: app/assets/javascripts/availability.js.coffee [hunk 1]
```diff
@@ -13,7 +13,8 @@ class AvailabilityUpdater
     'Remote Storage Request', 'Hold Request', 'Recall Request']
   missing_statuses = ['Missing', 'Lost--Library Applied',
```
Comment (kevinreiss) [inline app/assets/javascripts/availability.js.coffee]: Is it worth pulling these statuses from a config file rather than hardcoding them?

**Comment by tampakis** [comment]
I would be fine working with es6 over coffeescript. The problem is that I really don't know any flavor of js, and was just expanding on the code that was already in Orangelight. I also don't know how to load config files.

**Comment by kevinreiss** [comment]
I think we can tag those both as future issues to look at.

**Pull request merged**